models/labels/addresses/__single_category_labels__/balancer_v2/labels_balancer_v2_gauges.sql

```diff
@@ -1,4 +1,6 @@
-{{config(alias = alias('balancer_v2_gauges'),
+{{config(
+        tags=['prod_exclude', 'dunesql'],
```

Comment:
remind me, why `prod_exclude` tag, when it's not on spark version?

Reply:
this applies to all upstream spells in here too

Reply:
Sorry at the time I translated these spells, they are `prod_exclude` in spark spells, due to invalid sources.
I will update the tag